config/settings.yml

```diff
@@ -105,26 +105,33 @@ check_in:
     retry_attempt_expiry: 604800
   chip_api_v2:
     base_path: <%= ENV['check_in__chip_api_v2__base_path'] %>
-    host: <%= ENV['check_in__chip_api_v2__host'] %>
+    base_path_v2: <%= ENV['check_in__vaec_cie__chip__base_path'] %>
     mock: <%= ENV['check_in__chip_api_v2__mock'] %>
     redis_session_prefix: check_in_chip_v2
     service_name: CHIP-API
     timeout: 30
     tmp_api_id: <%= ENV['check_in__chip_api_v2__tmp_api_id'] %>
+    tmp_api_id_v2: <%= ENV['check_in__vaec_cie__chip__api_id'] %>
     tmp_api_user: <%= ENV['check_in__chip_api_v2__tmp_api_user'] %>
+    tmp_api_user_v2: <%= ENV['check_in__vaec_cie__chip__password'] %>
     tmp_api_username: <%= ENV['check_in__chip_api_v2__tmp_api_username'] %>
+    tmp_api_username_v2: <%= ENV['check_in__vaec_cie__chip__username'] %>
     url: <%= ENV['check_in__chip_api_v2__url'] %>
+    url_v2: <%= ENV['check_in__vaec_cie__chip__url'] %>
   lorota_v2:
     api_id: <%= ENV['check_in__lorota_v2__api_id'] %>
+    api_id_v2: <%= ENV['check_in__vaec_cie__lorota__api_id'] %>
     api_key: <%= ENV['check_in__lorota_v2__api_key'] %>
+    api_key_v2: <%= ENV['check_in__vaec_cie__lorota__api_key'] %>
     base_path: <%= ENV['check_in__lorota_v2__base_path'] %>
-    host: <%= ENV['check_in__lorota_v2__host'] %>
+    base_path_v2: <%= ENV['check_in__vaec_cie__lorota__base_path'] %>
```

Comment:
There are [values](https://us-gov-west-1.console.amazonaws-us-gov.com/systems-manager/parameters/?region=us-gov-west-1&tab=Table#list_parameter_filters=Name:Contains:check_in%2Fvaec_cie%2Florota%2Fbase_path) in the param store for dev, staging, & sandbox, did you want one for prod?

Reply:
@stiehlrod There're actually values in there for all four!

If you click the right angle bracket:

<img width="127" height="53" alt="Screenshot 2025-08-04 at 2 49 29 PM" src="https://github.com/user-attachments/assets/7419613d-4819-4157-a75a-d2c4096fea8d" />

The UI will refresh and you'll see all of the items, not just three.

Why AWS did this to us is not known, but it's a recent addition to my **List of Grievances** 🙂

Reply:
> Why AWS did this to us is not known

@ndouglas I actually created a support ticket with them about this 😆 and they said it's because we have too many parameters. So they didn't consider it a bug on their end 😒 

> a recent addition to my **List of Grievances**

SAME 😑